Update the introduction of open and closed types. (#1447)
dotnet/csharpstandard: standard/types.md

### 8.4.3 Open and closed types

> All types can be classified as either ***open types*** or ***closed types***. An open type is a type that involves type parameters. More specifically:

Author: The “can be” here should be something like “are” – we are defining things here. #1441 changes this line to:

```suggestion
Any type can be classified as either an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

This still says “can be” so I think it is wrong. It also reads wrong to me, but that is subjective. I’m going to make some alternative suggestions, each in a seperate comment. Please 👍/👎🏻 as you see fit, including this one, and whomever merges can OK the preferred choice, or we don't merge and the #1441 version and “can be” stays.

Author: ```suggestion
A type is classified as an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

Author: Is “Every” better than “A”?

```suggestion
Every type is classified as an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

Author: Is “classified” really required?

```suggestion
A type is either an ***open type*** or a ***closed type***. An open type is a type that involves type parameters. More specifically:
```

Author: The last one feels awkward, this seems better but loses ***defined terms***, @RexJaeschke feel free to correct my use of *italics* in lieu!

```suggestion
Every type is either **open** or **closed**. An open type is a type that involves type parameters. More specifically:
```

Author: Well those are my suggestions, all to remove “can be” – types *are* open/closed. Feel free to add your own of course :-)

Reviewer: I'm OK with most of the alternatives.

Reviewer 2: I prefer "A type is either an ***open type*** or a ***closed type***. " for 2 reasons:

1. When defining a term it's best to use the singular.
2. I prefer "type" as part of the term being defined, as without it the terms ***open*** and ***closed*** are just adjectives with no context.

Author: With @RexJaeschke’s 👍 I think we have a winner (3 votes counting Rex), unless there are some late votes…
